tests/tests/test_contacts/test_mixing.py

```diff
@@ -0,0 +1,25 @@
+import phoebe
+import numpy as np
+
+
+def test_mixing(verbose=False):
+    if verbose:
+        print('phoebe.default_contact_binary()')
```

Comment:
switch print() to logger?

Reply:
I would say this instance can be removed entirely even